tox.toml

```diff
@@ -0,0 +1,32 @@
+requires = ["tox>=4"]
+min_version = "4.26.0"
+env_list = [
+    "py39",
+    "py310",
+    "py311",
+    "py312",
+    "py313",
+    "pypy3.9",
+    "lint",
+]
+# isolated_build = true
```

Comment:
I haven't found this flag [here](https://tox.wiki/en/latest/config.html).

The closest match is https://tox.wiki/en/latest/config.html#isolated_build_env .

Reply:
In tox 4 this option is always enabled - https://tox.wiki/en/latest/upgrading.html#removed-tox-ini-keys .